aip/general/0001.md

```diff
@@ -28,6 +28,25 @@ serve as the source of truth for API-related documentation at Google and the
 means by which API teams discuss and come to consensus on API guidance. AIPs
 are maintained as Markdown files in the [AIP GitHub repository][].
 
+## Scope
+
+It doesn't make sense for *every* API to follow the AIPs - not even all
+*Google* APIs. Some relevant factors to consider include:
+
+- Who is writing code to call the APIs? Some AIPs (e.g. [141](AIP-141)) include
+  requirements with an expectation of APIs being called from client libraries
+  written in diverse programming languages. If the API producer is confident
+  that the API only needs to be called from Google-authored code, some of those
+  requirements may not be relevant.
+- How is the client code deployed? When the set of deployed clients is fully
+  known and fully controlled (most importantly for server-to-server APIs), many
+  requirements around compatibility can be skipped. For example, if you
+  *absolutely know* that no code exists that will call an RPC, it's safe to
+  remove that RPC.
+- Is the API actually implementing an existing API specification that can't be
```

Comment:
This is a 3P API like S3 or OSS API/protocol like ssh, MySQL, Redis, Kafka, HBase, OpenAPI, etc.?

Reply:
Another example: package registry APIs.

https://cloud.google.com/artifact-registry/docs/docker/pushing-and-pulling

Reply:
Thanks, added examples.